docs/rfcs/rfc-019-optional-features.md

```diff
@@ -0,0 +1,82 @@
+# Optional features
+
+## Problem
+
+A Neo4j GraphQL library design introduced is to add operators directly in the schema for the supported fields.
+The number of operators and other features grows and with them the size of the `augmented schema`.
+A solution to reduce the size of the problem is to introduce the possibility to configure the environment with only the features necessary.
+
+## Proposed Solution (Config)
+
+The Config solution is to extend the `Neo4jGraphQLConfig` to support the list of features to be included with the `features` attribute.
+
+### Usage Examples
+
+```javascript
+  const neoSchema = new Neo4jGraphQL({
+        config: {
+            enableDebug: true,
+            features: {
+                filters: {
+                    String: {
+                        GT: true,
+                        LT: false,
+                    }
```

Comment:
So I have another suggestion here as I understand that this may be _too_ granular.

How about a config item `comparators` which contains each comparator for strings.

```suggestion
                    String: {
                        comparators: {
                            GT: true,
                            LT: false,
                        }
                    }
```

But then also allow it to accept a boolean value, which would allow you to disable all comparators. Could be the best of both worlds?

```
                    String: {
                        comparators: false
                    }
```

What do people think?